docs/classic/go.md

```diff
@@ -50,21 +50,23 @@ The observation shape is a function of the board size _N_ and has a shape of (N,
 |    2    | Player<br>_'`0`: white, `1`: black_                       |
 
-While rendering, the board coordinate system is [GTP](http://www.lysator.liu.se/~gunnar/gtp/).
+While rendering, the board coordinate system is [GTP](http://www.lysator.liu.se/~gunnar/gtp/). 
+
+
 
 #### Action Space
 
 Similar to the observation space, the action space is dependent on the board size _N_.
 
-|   Action ID   | Description                                                                                                   |
-|:-------------:|---------------------------------------------------------------------------------------------------------------|
-|    0 ~ N-1    | Place a stone on the 1st row of the board.<br>_`0`: (0,0), `1`: (0,1), ..., `N-1`: (0,N-1)_                   |
-|    N ~ 2N-1   | Place a stone on the 2nd row of the board.<br>_`N`: (1,0), `N+1`: (1,1), ..., `2N-1`: (1,N-1)_                |
-|      ...      | ...                                                                                                           |
-| N^2-N ~ N^2-1 | Place a stone on the Nth row of the board.<br>_`N^2-N`: (N-1,0), `N^2-N+1`: (N-1,1), ..., `N^2-1`: (N-1,N-1)_ |
-|      N^2      | Pass                                                                                                          |
+|                          Action ID                           | Description                                                  |
+| :----------------------------------------------------------: | ------------------------------------------------------------ |
+| <img src="https://render.githubusercontent.com/render/math?math=0 \ldots (N-1)"> | Place a stone on the 1st row of the board.<br>_`0`: (0,0), `1`: (0,1), ..., `N-1`: (0,N-1)_ |
+| <img src="https://render.githubusercontent.com/render/math?math=N \ldots (2N- 1)"> | Place a stone on the 2nd row of the board.<br>_`N`: (1,0), `N+1`: (1,1), ..., `2N-1`: (1,N-1)_ |
+|                             ...                              | ...                                                          |
+| <img src="https://render.githubusercontent.com/render/math?math=N^2-N \ldots N^2-1"> | Place a stone on the Nth row of the board.<br>_`N^2-N`: (N-1,0), `N^2-N+1`: (N-1,1), ..., `N^2-1`: (N-1,N-1)_ |
+| <img src="https://render.githubusercontent.com/render/math?math=N^2"> | Pass                                                         |
 
-For example, you would use action `4` to place a stone on the board at the (0,3) location or action `N^2` to pass. You can transform a non-pass action `a` back into its 2D (x,y) coordinate by computing `(a//N, a%N)` The total action space is N^2+1.
+For example, you would use action `4` to place a stone on the board at the (0,3) location or action `N^2` to pass. You can transform a non-pass action `a` back into its 2D (x,y) coordinate by computing `(a//N, a%N)` The total action space is <img src="https://render.githubusercontent.com/render/math?math=N^2 %2B 1">.
```

Comment:
why are there hard coded image refs like this here and above?

Reply:
Oh, I replaced the text equations with generated images of the rendered equations, so instead of N^2-1 you get actual superscripts and such. The equations there were annoying to read so I thought it helped